Spigot-Server-Patches/0616-Fixed-crossbows-firing-ammo-when-the-event-is-cancel.patch

```diff
@@ -0,0 +1,40 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Cameron <cjpuffnstuff@gmail.com>
+Date: Wed, 16 Dec 2020 18:02:36 -0800
+Subject: [PATCH] Fixed crossbows firing ammo when the event is cancelled
+
+
+diff --git a/src/main/java/net/minecraft/server/ItemCrossbow.java b/src/main/java/net/minecraft/server/ItemCrossbow.java
+index 14c0e7382292b3d39858d4d957df8016c301c712..d7045abfe9ec943a2209c1c4bddd95a2cf6f5957 100644
+--- a/src/main/java/net/minecraft/server/ItemCrossbow.java
++++ b/src/main/java/net/minecraft/server/ItemCrossbow.java
+@@ -30,7 +30,6 @@ public class ItemCrossbow extends ItemProjectileWeapon implements ItemVanishable
+ 
+         if (d(itemstack)) {
+             a(world, entityhuman, enumhand, itemstack, m(itemstack), 1.0F);
+-            a(itemstack, false);
```

Comment:
please don't remove this line and the l one, comment them out instead, saying they got moved.